test/suite_test.go

```diff
@@ -90,9 +91,10 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme:             scheme.Scheme,
-		MetricsBindAddress: "0",
-		Port:               9443,
+		Scheme: scheme.Scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: "0",
```

Comment:
By same reasoning as above.
```suggestion
			BindAddress: "0:9443",
```

Reply:
0 is a magic value to disable the metrics server. see https://github.com/search?q=repo%3Akubernetes-sigs%2Fcontroller-runtime%20metricsserver.Options&type=code